Geometry/MuonNumbering/src/MuonBaseNumber.cc

```diff
@@ -1,7 +1,7 @@
 #include "Geometry/MuonNumbering/interface/MuonBaseNumber.h"
 #include "FWCore/MessageLogger/interface/MessageLogger.h"
 
-//#define LOCAL_DEBUG
+#define LOCAL_DEBUG
```

Comment:
@slomeo Please explain this change. It turns on the debug statements. The problem in #28796 was that the debug statements were effectively turned on. I think you want to turn OFF the debug statements, which means this line should remain commented out. Am I missing something?

Reply:
@cvuosalo : The change is made to answer to @fabiocos  (see #28796)

Reply:
@slomeo But how does it answer Fabio's request? It seems to do the opposite of what he requested.

Reply:
@cvuosalo I could make a test: restore the version where only the line 4 is commented (i.e //#define LOCAL_DEBUG)

Reply:
Yes, as Fabio suggested, please make a version where only line 4 is commented.
It should appear like the left side of https://github.com/cms-sw/cmssw/pull/28796/files/a6175cdc9b547560bede5f274e12ca51e431d5dd#diff-47cb6cf59007852c622e5f745bab812dL4-L42

Reply:
@slomeo I am afraid you misinterpreted my suggestion, we want to silence the messages but keep the possibility to switch them on, not to enable them by default